app/views/users/show.html.erb

```diff
@@ -1,28 +1,30 @@
-<div>
-  <section>
-    <div>User Image</div>
-    <div>
-      <h1><a><%= @user.name %></a></h1>
-      <span>Number of posts:<%= @user.posts_counter %></span>
+<main>
+  <section class="user-card">
+    <div class="user-image"><%= image_tag @user.photo %></div>
+    <div class="card-details">
+      <h3><a><%= @user.name %></a></h3>
+      <span>Number of posts: <%= @user.posts_counter %></span>
     </div>
   </section>
 
-  <section>
+  <section class="bio-card">
     <h2>Bio</h2>
     <p><%= @user.bio %></p>
   </section>
 
-  <section>
-   <ul>
-     <% @user.posts.each do |post| %>
-       <li>
-         <h3>Post <%='#'%><%= post.id %></h3>
-         <p>post.text</p>
-         <span>Comments: <%= post.comments_counter %></span>
-         <span>Likes: <%= post.likes_counter %></span>
-       </li>
-     <% end %>
-   </ul>
-   <button>See all posts</button>
+  <section class="posts-sec">
+    <h2>Recent Posts</h2>
+    <ul>
+      <% @user.posts.each do |post| %>
+        <%= link_to user_post_path(@user.id, post.id) do %>
+          <li class="post-card">
+            <h3>Post <%='#'%><%= post.id %></h3>
+            <p><%= post.text %></p>
+            <span>Comments: <%= post.comments_counter %> Likes: <%= post.likes_counter %></span>
+          </li>
+        <% end %>
+      <% end %>
+    </ul>
+    <button><%= link_to user_posts_path(@user.id) do %>See all posts<% end %></button>
```

Comment:
- [ ] As requested by he previous reviewer kindly align the `see all posts` button as it is in the given design

Your design
![image](https://user-images.githubusercontent.com/81932478/206863822-5c201519-b7ac-422a-afb1-852bba6f2bcb.png)

Given Wireframe
![image](https://user-images.githubusercontent.com/81932478/206863861-798a4dbd-7f01-4021-a538-dc6cfe018b3c.png)